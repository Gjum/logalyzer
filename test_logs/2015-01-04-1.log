[04:00:00] [Server thread/INFO]: [Server] crash, was still on: XXX
[04:00:00] [Server thread/INFO]: [Server] Online  Ulexos 07+48 -
[04:00:00] [Server thread/INFO]: [Server] Online  Offlinegott 08+48 -