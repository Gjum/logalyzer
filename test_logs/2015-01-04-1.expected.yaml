first_event: 1420340400
last_event: 1420340400
online:
  Offlinegott: [9f352de0-b533-425c-9435-89b918ae4602, 1420095600, 1]
  Ulexos: [a1d94087-e9c0-412f-b90f-cff2c224d44f, 1420171200, 1]
started: false
stopped: null
times: []
